app/Http/Controllers/GetPendingSubmissionsController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Enums\SubmissionStatuses;
+use App\Models\Submission;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
+
+class GetPendingSubmissionsController
+{
+    /**
+     * Handle the incoming request.
+     *
+     *
+     * @return \Illuminate\Http\Response
```

Comment:
```suggestion
     * @return LengthAwarePaginator
```
Yo en lo personal sacaria el docblock igual.